pkgs/tools/security/sshguard/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchgit, autoreconfHook, yacc, flex, pkgs}:
+
+
+stdenv.mkDerivation rec {
+  version = "1.7.1";
+  name = "sshguard-${version}";
+
+  src = fetchgit {
+    url = "https://bitbucket.org/sshguard/sshguard.git";
+    rev = "8600fd5b70cd80fad87c414fd7c410082991a209";
+    sha256 = "1i2ic8nca4f61pl2ggjp4x4an37hhvjgk2wv4smjsj6lvz3bw06a";
+  };
+
+  doCheck = true;
+
+  nativeBuildInputs = [ autoreconfHook yacc flex ];
+
+  configureFlags = [ "--with-firewall=iptables" ];
+
+  postInstall = ''
+      cat > "$out/sbin/sshguard-journalctl" << EOF
+      #!/bin/sh
+      SSHGUARD_OPTS=\$1
+      shift
+      LANG=C journalctl -afb -p info -n1 -o cat "\$@" | $out/bin/sshguard \$SSHGUARD_OPTS
+      EOF
+      chmod +x "$out/sbin/sshguard-journalctl"
+    '';
+
+  meta = with stdenv.lib; {
+    description = "SSHGuard protects hosts from brute-force attacks";
+    longDescription = ''
+      SSHGuard can read log messages from various input sources. Log messages are parsed, line-by-line, for recognized patterns.
+      If an attack, such as several login failures within a few seconds, is detected, the offending IP is blocked.
+    '';
+    homepage = https://sshguard.net;
+    license = stdenv.lib.licenses.bsd3;
+    maintainers = [ maintainers.sargon ];
+    platforms = stdenv.lib.platforms.all;
```

Comment:
You don't need to rewrite `stdenv.lib` for licence and platforms.
Also on platforms : you are explicitly building for `iptables` backend so it won't work on Darwin.